contracts/truefi2/interface/ILoanToken2.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {ITrueFiPool2} from "./ITrueFiPool2.sol";
+import {ILoanToken} from "../../truefi/interface/ILoanToken.sol";
+
+interface IContractWithPool {
+    function pool() external view returns (ITrueFiPool2);
+}
+
+// Had to be split because of multiple inheritance problem
+interface ILoanToken2 is ILoanToken, IContractWithPool {
```

Comment:
Should we just create a new LoanToken interface? It might be harder to read / understand the inheritance, and we are Deprecating the old loan token anyways.

Reply:
There is still active development of current LoanToken and it's not going to change much. So let's keep it like that to avoid manual merging of changes in old LoanToken into the new one. And later we can make it a single contract.